### PR TITLE
Support LTS .Net Core 3.1

### PR DESCRIPTION
**Pull request by eopeter**
Upgraded Project References to Support LTS .Net Core 3.1

### CODE REVIEW

**Comment by eopeter** [comment]
it seems the SDK 3.1.100 is not yet available in Azure hosted tool cache.

**Comment by nabinked** [comment]
Thanks for the PR. Will review and test this soon

**Comment by nabinked** [comment]
Can you update the dotnet version here https://github.com/nabinked/NToastNotify/blob/1d8805ce34ca7abfa8f12927689ba488dfef4ce4/azure-pipelines.yml#L27
and see if the builds pass ?

**Comment by eopeter** [comment]
That did it! Thanks

File: src/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
[the remark this answers is not in the document]
Comment (eopeter) [reply src/Properties/launchSettings.json]: That is true. I removed it.

**Pull request merged**